assets/js/modules/adsense/datastore/urlchannels.test.js

```diff
@@ -127,22 +125,44 @@ describe( 'modules/adsense URL channels', () => {
 						{ status: 500 }
 					);
 
-				const fakeAccountID = 'pub-777888999';
 				const fakeClientID = 'ca-pub-777888999';
 
 				muteConsole( 'error' );
-				registry.select( STORE_NAME ).getURLChannels( fakeAccountID, fakeClientID );
+				registry.select( STORE_NAME ).getURLChannels( fakeClientID );
 				await subscribeUntil( registry,
 					// TODO: We may want a selector for this, but for now this is fine
 					// because it's internal-only.
-					() => store.getState().isFetchingURLChannels[ `${ fakeAccountID }::${ fakeClientID }` ] === false,
+					() => store.getState().isFetchingURLChannels[ fakeClientID ] === false,
 				);
 
 				expect( fetch ).toHaveBeenCalledTimes( 1 );
 
-				const urlchannels = registry.select( STORE_NAME ).getURLChannels( fakeAccountID, fakeClientID );
+				const urlchannels = registry.select( STORE_NAME ).getURLChannels( fakeClientID );
 				expect( urlchannels ).toEqual( undefined );
 			} );
+
+			it( 'dispatches an error when account ID cannot be parsed from client ID', async () => {
+				const clientResponse = {
```

Comment:
Seems like this should be mocking a response rather than doing it client-side (see my comment above).